docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Integration with cert-manager"
+description: "Learn how to use cert-manager certificates with Traefik Proxy for your routers. Read the technical documentation."
+---
+
+# cert-manager
+
+Provision TLS Certificate for Traefik Proxy with cert-manager on Kubernetes
+{: .subtitle }
+
+Traefik Proxy can get Certificates from cert-manager on Kubernetes.
```

Comment:
Not sure that this sentence is needed, it seems to be a repetition of the previous one.

Reply:
I double checked and you're right.

Reply:
It's fixed.